gramps/gen/filters/rules/place/_withinarea.py

```diff
@@ -106,17 +116,19 @@ def prepare(self, db, user):
                 self.radius = float(value)
             self.radius = self.radius / 2
 
-    def apply(self, dummy_db, place):
+    def apply_to_one(self, db: Database, place: Place) -> bool:
+        latit: Union[float, None] = None
+        longit: Union[float, None] = None
         if self.handle is None:
             return False
         if self.latitude is None:
             return False
         if self.longitude is None:
             return False
         if place:
-            lat = place.get_latitude()
-            lon = place.get_longitude()
-        if lat and lon:
+            lat = place.lat
+            lon = place.long
+        if lat is not None and lon is not None:
```

Comment:
I assume a bugfix for when `lat` and `lon` are both zero

Reply:
I think I should put the logic back... you are right that it could change the result and I didn't mean to "fix" anything.